components/wb-bubble/_base.scss

```
@@ -0,0 +1,103 @@
+
+/*
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * Bubble - Stable CSS
+ */
+/* Animations */
+
+@keyframes slideInFromRight {
+	0% {
+		transform: scale(0, 1); }
+	95% {
+		transform: scale(0, 1); }
+	100% {
+		transform: scale(1, 1); } }
+
+@keyframes pulseIn {
+	0% {
+		transform: scale(1, 1); }
+	15% {
+		transform: scale(1.15, 1.15); }
+	30% {
+		transform: scale(1, 1); }
+	65% {
+		transform: scale(1.3, 1.3); }
+	100% {
+		transform: scale(1, 1); } }
+
+.trans-left {
+	animation-delay: 0s;
+	animation-duration: 5s;
+	animation-iteration-count: 1;
+	animation-name: slideInFromRight;
+	animation-timing-function: ease-out;
+	transform-origin: 100% 50%;
+	will-change: scroll-position; }
```

put the `}` on the next line.
```suggestion
	will-change: scroll-position; 
}
```